docs/_ralph-wiggum/protocol.md

```
@@ -105,21 +105,22 @@ git pull origin main
-git checkout -b ralph-wiggum-loop  # if you skipped dev, create this off main
+# Use a repo-scoped name to avoid collisions across repos (recommended: kalshi-ralph).
+git checkout -b kalshi-ralph  # if you skipped dev, create this off main
 
 # Push branches to remote for backup
 git push -u origin dev  # optional
-git push -u origin ralph-wiggum-loop
+git push -u origin kalshi-ralph
 ```
 
 **Branch hierarchy (recommended):**
 ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)